Ginger/GingerCoreNET/ALMLib/ALMUserConfig.cs

```diff
@@ -64,6 +64,20 @@ public string ALMPassword
                 EncryptedALMPassword = EncryptionHandler.EncryptString(value, ref res);
             }
         }
+        private string mALMToken;
+        [IsSerializedForLocalRepository]
```

Comment:
Can we use the password field itself for token instead of new field ?

Reply:
i asked to add the token option and to keep the password option

Reply:
Yes. For sure we need to keep the password option as well. But what i am saying is to use the same field i,e, ALMPassword to store the token on XML. Because user can use only 1 option at a time

Also when running from command line we are using the password field. if we want to add a new field then need to add it also in CLI